gulpfile.js

```diff
@@ -27,8 +27,8 @@ gulp.task('sass-bootstrap', ['sass-sync'], function () {
     .pipe(gulp.dest('css/'));
 });
 
-gulp.task('sass-civicrm', ['sass-sync'], function () {
-  gulp.src('scss/civicrm/custom-civicrm.scss')
+gulp.task('sass:civicrm', ['sass:sync'], function () {
+  return gulp.src('scss/civicrm/custom-civicrm.scss')
```

Comment:
i think the hanging was due to how massive the custom-civicrm.css file is (it relies on way to  many `@extends` at the moment).

What `return` does is to allow gulp to understand when the task has effectively done running, otherwise without it you would get something like this
```js
[08:15:33] Starting 'sass:bootstrap'...
[08:15:33] Finished 'sass:bootstrap' after 13 ms
[08:15:33] Starting 'sass:civicrm'...
[08:15:33] Finished 'sass:civicrm' after 2.8 ms
[08:15:33] Starting 'sass'...
[08:15:33] Finished 'sass' after 2.81 μs
```
instead of this
```js
[08:14:41] Starting 'sass:bootstrap'...
[08:14:41] Starting 'sass:civicrm'...
[08:14:50] Finished 'sass:bootstrap' after 8.79 s
[08:15:03] Finished 'sass:civicrm' after 22 s
[08:15:03] Starting 'sass'...
[08:15:03] Finished 'sass' after 9.94 μs
```
notice that in the first case gulp reports `sass:civicrm` to be done after 3 ms, but in reality sass is still processing the files